docs/pbc/all/cart-and-checkout/202410.0/base-shop/feature-overviews/cart-feature-overview/dynamic-cart-overview.md

```diff
@@ -15,6 +15,11 @@ The following video shows how a user can interact with different elements of a c
   </video>
 </figure>
 
+## Configuration recommendation
+
+For good rendering performance, disable cart validation during rendering by setting `CartPageConfig::IS_QUOTE_VALIDATION_ENABLED_FOR_AJAX_CART_ITEMS = false`.
```

Comment:
Note: this config appears only in 202505.0, see `_includes/pbc/all/install-features/202505.0/install-the-cart-feature.md`

Reply:
@matweew Then you need to add this page to 202505 and add this content there